src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -396,12 +454,38 @@ case class Block(center: (Double,Double,Double), faces: Seq[Face]) {
   }
 
   /**
+    * Calculates the volume of the block
```

Comment:
Could you provide a reference to the method you're using to compute volume here in the comments? Something as simple as a link would suffice.